src/main/java/dk/dataforsyningen/vanda_hydrometry_data/command/WaterLevelsCommand.java

```diff
@@ -113,14 +113,18 @@ public int saveData() {
 	@Override
 	public void displayData(boolean raw) {
 		if (raw && data != null) {
-			VandaHUtility.logAndPrint(null, null, config.isVerbose(), "Number of items: " + data.length);
+			if (config.isVerbose()) {
+    			System.out.println("Number of items: " + data.length);
+			}
 			for(DmpHydroApiResponsesMeasurementResultResponse item : data) {
 				System.out.println(item);
 			}
 		}
 		if (!raw && measurements != null && measurementTypes != null) {
-			VandaHUtility.logAndPrint(null, null, config.isVerbose(), "Number of measurements: " + measurements.size());
-			VandaHUtility.logAndPrint(null, null, config.isVerbose(), "Number of measurementTypes: " + measurementTypes.size());
+			if (config.isVerbose()) {
+    			System.out.println("Number of measurements: " + measurements.size());
+    			System.out.println("Number of measurementTypes: " + measurementTypes.size());
+			}
```

Comment:
ændre dette til
`logger.info("Number of measurements: " + measurements.size());`
`logger.info("Number of measurementTypes: " + measurementTypes.size());`